src/db/QDjangoQuerySet.h

```diff
@@ -451,6 +452,20 @@ int QDjangoQuerySet<T>::count() const
     return query.value(0).toInt();
 }
 
+/** Counts the objects in the queryset using an SQL [AVG, COUNT, SUM, MIN, MAX] query,
+ *  or -1 if the query failed.
+ *
+ */
+template <class T>
+QVariant QDjangoQuerySet<T>::aggregate(const QString& field, QDjangoWhere::AggregateType func) const
+{
+    // execute aggregate query
+    QDjangoQuery query(d->aggregateQuery(field,func));
+    if (!query.exec() || !query.next())
+        return -1;
```

Comment:
Not sure about using -1 as the error, there are legitimate results which will also be -1. Any other ideas?